requirements.txt

```diff
@@ -50,6 +50,7 @@ urllib3==1.26.5
 oauthlib==2.0.6
 requests-oauthlib==0.8.0
 raven==6.4.0
+django-redis==5.2.0
```

Comment:
Wondering any special notes for choosing this version? Is this version limited by our Python and Django versions (3.6 and 3.2 respectively)?

Reply:
It is their latest version.

Reply:
Yep. It supports Python 3.6 and Django 3.2: https://pypi.org/project/django-redis/5.2.0/

![Screen Shot 2023-03-03 at 1 04 18 PM](https://user-images.githubusercontent.com/3750414/222794978-5cb22aba-8885-4186-a610-7ecd56d7ff9d.png)